assets/src/edit-story/components/panels/layer/layer.js

```diff
@@ -121,8 +124,8 @@ function Layer({ layer }) {
         {isBackground ? (
           <LockedIcon aria-label={__('Background element', 'web-stories')} />
         ) : (
-          <LayerIcon />
-        )}
+            <LayerIcon />
+          )}
```

Comment:
The linter complains about those two changed lines here.

Reply:
Fixed, I accidentally used a different formatter than Prettier.